pkg/config/workloads.go

```diff
@@ -135,14 +159,35 @@ func (w Workloads) workloadGivenActivationAnnotation(sboxAnnotations map[string]
 	return nil
 }
 
+// milliCPUToQuota converts milliCPU to CFS quota and period values.
+// Input parameters and resulting value is number of microseconds.
+func milliCPUToQuota(milliCPU, period int64) (quota int64) {
```

Comment:
nit: can functions be defined below they're called?

Reply:
Yeah, absolutely, updating